tests/test_warning.py

```diff
@@ -101,7 +97,7 @@ def test_full_with_coastal_bulletint(dep: str, res: bool) -> None:
     full_phenomenoms = client.get_warning_full(domain=dep, with_costal_bulletin=True)
 
     has_coastal_phenomenom = any(
-        phenomenom["phenomenon_id"] == 9
+        phenomenom["phenomenon_id"] == "9"
```

Comment:
mypy issue here, maybe still an int ?

Reply:
I updated my code. The phenomenon id returned by meteo france is now a character string. There should be no more typing errors.